src/views/catalog/customize/CustomizeVirtualMachine.tsx

```diff
@@ -55,8 +66,9 @@ const CustomizeVirtualMachine: React.FC = () => {
             <RightHeader template={template} />
           </div>
           <div className="col-md-5 col-md-pull-7">
-            {template && loaded && <Form template={template} />}
-            {!loaded && <CustomizeVirtualMachineSkeleton />}
+            {template && loaded && loadedSources && bootSourcesLoaded && (
```

Comment:
This is redundant. if we have the condition at [line 57 ](https://github.com/kubevirt-ui/kubevirt-plugin/pull/872/files#diff-6c413ef7479fee94773100ffaf6300a63d4aa5862658371a2a2c73297a007b42R57), you can simply do `{template && (Component)}`